### DISCUSSION TITLE
help with including a favicon

### DISCUSSION THREAD

**Discussion by el-chinosauro** [alshedivat/al-folio#2606] [Q&A] [answered]
Hi, and thanks for a fantastic theme. I have tried without luck to include a favicon to my website. Could you please point me to the correct way of doing so? thanks!

**Comment by george-gca** [comment, marked as answer]
You can set it in the icon part in [_config.yml](https://github.com/alshedivat/al-folio/blob/e14f5723f2ca14ee15f8e1f65f07477f5d4485af/_config.yml#L18).
